en/04_Changelogs/5.0.0.md

```diff
@@ -100,6 +130,15 @@ This is a major release and as a result there are a number of breaking API chang
 - `SilverStripe\Core\Extensible::invokeWithExtensions()` and `SilverStripe\Core\Extensible::extend()` now use the splat operator instead of having a concrete number of possible arguments.
 - The constructor for `SilverStripe\Versioned\Versioned` now explicitly only accepts mode as a single argument.
 
+### Email {#api-email}
+
+- Email was sent in CMS 4 using [SwiftMailer](https://swiftmailer.symfony.com/docs/introduction.html). In CMS 5 this has been replaced with [symfony/mailer](https://symfony.com/doc/current/mailer.html).
+- `symfony/mailer` is the currently maintained email package from Symfony, and it also a more flexible email system that allows easier integration with third-party email providers
+- Previously in CMS 4 there was official support for both `SendmailTransport` that used the sendmail binary available on most webservers, as well as `MailTransport` which uses the native php function `mail()`.
+- Older Silverstripe project were configured to use `MailTransport` out of the box. This was changed to `SendmailTransport` newer projects created using `silverstripe/installer` `4.9.0` or greater. `MailTransport` was added in a `thirdparty` folder in CMS 4 to retain support after it was officially removed in later versions of `SwiftMailer`.
```

Comment:
I don't think this background information is necessary - lets just mention what has changed from 4 to 5 rather than what changed from 4 to another version of 4
```suggestion
```

Reply:
Remooved